RELEASE.md

```diff
@@ -62,7 +62,12 @@ Bump version
     $ sed -r -i "s/TOPOLVM_VERSION := [[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+/TOPOLVM_VERSION := ${VERSION}/g" example/Makefile
     $ sed -r -i "s/checkout v[[:digit:]]+\.[[:digit:]]+\.[[:digit:]]+/checkout v${VERSION}/g" example/README.md
     ```
-4. Edit `CHANGELOG.md` for the new version ([example][]).
+4. Edit `CHANGELOG.md` for the new version ([example][]).  
+   The list of relevant PRs can be obtained by the following command.
```

Comment:
The result may contain a PR with changes only for the helm chart. We need to filter out such PRs.

Reply:
Thought it is ideal to filter out helm-only PRs, I think it is hard to realize (at least I could not come up with any simple ideas to do it).
In the current release procedure, we first make a list of all PRs an then filter and categorize each item manually. In this PR I aimed to automate the listing process. I would like to think the filtering and categorization as another problem.

I changed the explanation to make it clear that the command's output is just candidates, and we need a filtering process.

Reply:
I agreed. It would be sufficient to filter out manually.